packages/babel-helper-create-class-features-plugin/src/fields.ts

```diff
@@ -206,12 +206,30 @@ function unshadow(
   }
 }
 
+export function buildCheckInRHS(
+  rhs: t.Expression,
+  file: File,
+  inRHSIsObject?: boolean,
+) {
+  let helperRef;
+  try {
+    helperRef = file.addHelper("checkInRHS");
+  } catch (err) {
+    if (err.code !== "BABEL_HELPER_UNKNOWN") {
```

Comment:
Can we use `availableHelper`?

```js
if (inRHSIsObject || !file.availableHelper?.("checkInRHS")) return rhs;
return t.callExpression(file.addHelper("checkInRHS"), [rhs]);
```

this also avoids injecting the helper when `inRHSIsObject` is false.